### ISSUE TITLE
We should close this | edit: should we redirect this to the wiki?

### ISSUE THREAD

**Issue by kylebakerio** [aframevr/aframe-registry#91]
It makes aframe look dead. Just turn it into the equivalent of a subreddit if nothing else. Preferably make it a wiki so the community can maintain it. Almost anything would be better than it sitting here as a monument to 2016/2017. A month into an A-frame project, I still can't shake the feeling that aframe is dying because almost every example I look up is old, and this created that strong first impression.

**Comment by kylebakerio** [comment]
Just a heads up to others who end up here: I went ahead and made that wiki eventually, and at this point I would say it is a success. See: https://aframe.wiki/component-directory

That said, I would hope/suggest:
1) We redirect this URL to that page for the short term.
2) Longer term, I'd be interested in updating this to dynamically reflect that page with its current format, or something along those lines. The presentation in the wiki could be improved--it's very utilitarian right now, which is great (and maintainable), but could be better.

Open to other spin-offs of this idea, but I do think this website sitting around in its current form, especially since it's still pointed to in the docs, is likely hurting A-Frame's ability to retain new users.

**Comment by donmccurdy** [comment]
Related:

- https://github.com/aframevr/aframe-registry/issues/80
- https://github.com/aframevr/aframe-registry/issues/61

***

@kylebakerio thanks for bringing this up — yeah, certainly would prefer the list be maintained and up to date, that is more important than any web-appy browsing features. As a working hypothesis, I hope and expect a wiki will stay more evergreen, but ultimately it comes down to people maintaining the resource.

If @dmarcos agrees then I'm good with the idea of archiving this repository and pointing to a wiki. It would probably be best to have some redundant admin ownership of the wiki.

**Comment by kylebakerio** [comment]
@donmccurdy indeed. While I am hoping it is more evergreen, and it seems to be succeeding at that right now (I left it for 6 months and came back to find it more successful than when I left, and features 4 editors in the last month), at the very least it is more up to date than the registry is now--and when I started aframe a few years ago, the registry was already deeply out of date and that already had me wondering if A-Frame was dead. I really wonder how many people start there and then move on to other options without digging much further during a preliminary evaluation of the space.

Happy to add other admins. Would also be ideal if I wasn't the sole owner of the URL.
